app/stylesheets/Free.scss

```diff
@@ -1,4 +1,4 @@
-.tilt-icon.tilt-icon-free {
+.tilt-icon.tilt-icon-tag-free {
```

Comment:
@cohitre Are the tilt-images stylesheets used for anything other than styling the images served at the local express server? This change wires up the Free icon to the styles declared in Free.scss.

![image](https://cloud.githubusercontent.com/assets/590608/11124109/1b0271fc-8918-11e5-90b0-faa4f5a58db7.png)


![](https://files.slack.com/files-pri/T02GBQTLU-F0EB8B34L/pasted_image_at_2015_11_11_03_41_pm.png)


![image](https://cloud.githubusercontent.com/assets/590608/11124119/2a56aede-8918-11e5-98a5-50fa1c1f3d74.png)